ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -275,36 +254,36 @@ public void testPrint() throws InterruptedException {
 
   @Test
   public void testPropertySetUnset() {
-    test("set 'application.id' = 'Test_App'", EMPTY_RESULT);
-    test("set 'producer.batch.size' = '16384'", EMPTY_RESULT);
-    test("set 'max.request.size' = '1048576'", EMPTY_RESULT);
-    test("set 'consumer.max.poll.records' = '500'", EMPTY_RESULT);
-    test("set 'enable.auto.commit' = 'true'", EMPTY_RESULT);
-    test("set 'ksql.streams.application.id' = 'Test_App'", EMPTY_RESULT);
-    test("set 'ksql.streams.producer.batch.size' = '16384'", EMPTY_RESULT);
-    test("set 'ksql.streams.max.request.size' = '1048576'", EMPTY_RESULT);
-    test("set 'ksql.streams.consumer.max.poll.records' = '500'", EMPTY_RESULT);
-    test("set 'ksql.streams.enable.auto.commit' = 'true'", EMPTY_RESULT);
-    test("set 'ksql.service.id' = 'test'", EMPTY_RESULT);
-
-    test("unset 'application.id'", EMPTY_RESULT);
-    test("unset 'producer.batch.size'", EMPTY_RESULT);
-    test("unset 'max.request.size'", EMPTY_RESULT);
-    test("unset 'consumer.max.poll.records'", EMPTY_RESULT);
-    test("unset 'enable.auto.commit'", EMPTY_RESULT);
-    test("unset 'ksql.streams.application.id'", EMPTY_RESULT);
-    test("unset 'ksql.streams.producer.batch.size'", EMPTY_RESULT);
-    test("unset 'ksql.streams.max.request.size'", EMPTY_RESULT);
-    test("unset 'ksql.streams.consumer.max.poll.records'", EMPTY_RESULT);
-    test("unset 'ksql.streams.enable.auto.commit'", EMPTY_RESULT);
-    test("unset 'ksql.service.id'", EMPTY_RESULT);
-
-    testListOrShow("properties", build(validStartUpConfigs()), false);
+    testRunner.test("set 'application.id' = 'Test_App'", EMPTY_RESULT);
```

Comment:
minor comment - this could optionally call a test() method where it calls onto testRunner.test() - would remove some of the duplication....

Reply:
one improvement at a time, eh?

Reply:
lazy sod - less code is better code ;)